sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/metadata.yaml

```diff
@@ -12,8 +12,8 @@ description: |-
   effectively appeared on that date. The submission_date indicates
   when the server received the data.
 
-  The query for this table overwrites the whole table instead of writing to
-  a single partition, so manual backfills must use parameter --no_partition.
+  Note that the query for this table overwrites the whole table instead of writing to
```

Comment:
Could you add here what happens when using `bqetl backfill` for one random submission_date?  -- in relation to the changes in 4769?

Reply:
It should run as if the query was run in Airflow on that day. Depending on the `query.sql` and `metadata.yaml`